<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diskriminant</class>
 <widget class="QMainWindow" name="Diskriminant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../icons/dicon.png</normaloff>../../icons/dicon.png</iconset>
  </property>
  <property name="accessibleName">
   <string>Diskriminant</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 6px;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    border: 1px solid rgba(255, 246, 246,80);
    border-radius: 10px;
    font-family: Comic Sans MS;
    font-size: 16px;
    color: white;
    background-color: rgb(62, 75, 75)
}

QLabel{
    border-radius: 10px;
    font-family: Comic Sans MS;
    color:white;
    font-size: 16px;
    background-color: rgb(62, 75, 75);
    border:none;
} 

QWidget{
    background-color: rgb(22, 54, 56);
}

QPushButton{
    background-color: rgba(116, 63, 240,90);
    font-family: Comic Sans MS;
    font-size: 16px;
    border: 1px solid rgba(247,247,247,80);
    border-radius: 10px;
    color: white;
}

QPushButton:hover{
    opacity: 0.2;
    transition: all 0.5s;
}

QPushButton:pressed{
    background-color: rgba(116, 63, 240,90);
	border: 1px solid rgba(247,247,247,80);
}

QFrame{
    background-color: rgb(62, 75, 75);
    border-radius: 10px;
    border: none;
} 

QComboBox{
    font-family: Comic Sans MS;
    font-size: 16px;
    background-color:rgb(86, 83, 90);
    border-radius: 10px;
    border:1px solid rgb(113, 111, 117);
	color:white;
}

QPlainTextEdit{
	border: 1px solid rgba(247,247,247,80);
    border-radius: 10px;
    font-family: Comic Sans MS;
    font-size: 14px;
    color: white;
    background-color:rgb(86, 83, 90)
}</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User elections</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>a  =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>b  =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>c  =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditA">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditC">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Formulas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>D = b^2 - 4ac</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x1 = (-b - √D) / 2a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x2 = (-b + √D) / 2a</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelResult1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelResult2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelResult3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelResult4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="calculateButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
